### DISCUSSION TITLE
How to share my better notes workspace in Zotero Group?

### DISCUSSION THREAD

**Discussion by ZhouMaomaoZhou** [windingwind/zotero-better-notes#629] [Q&A] [answered]
As the title asked, I build a group in Zotero, and share my paper with others. Now, I want to share the workspace note with the group and let others edit it, is this feasible?

**Comment by windingwind** [comment, marked as answer]
create a note in the group and they will be able to read/edit it.

**Reply by ZhouMaomaoZhou** [reply]
That's helpful, thank you!
